modules/metastore/tests/src/Functional/MetastoreUpdatePathTest.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Drupal\metastore\Functional;
+
+use Drupal\FunctionalTests\Update\UpdatePathTestBase;
+
+/**
+ * Tests update functions for the metastore module.
+ *
+ * @group metastore
+ * @group update
+ */
+class MetastoreUpdatePathTest extends UpdatePathTestBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function setDatabaseDumpFiles() {
+    $this->databaseDumpFiles = [
+      __DIR__ . '/../../../../../tests/fixtures/update/update-2.20.0.php.gz',
+    ];
+  }
+
+  public function testUpdates8010on(): void {
```

Comment:
Test is self explanatory but a comment would be nice here so we know what the test is doing.